src/tanokInReact.js

```diff
@@ -63,7 +72,10 @@ export class TanokInReact extends React.Component {
     const state = this.state;
     return React.createElement(
       state.view,
-      Object.assign(state.state, {
+      Object.assign(
+        this.props.stateSerializer
+          ? this.props.stateSerializer(state.state)
```

Comment:
I don't think there are cases when stateSerializer must be changed on the fly, so we can save it in `this` in constructor